e2etests/bpfink.go

```diff
@@ -186,6 +194,10 @@ func (instance *BPFinkInstance) CheckIsHealthy(t *testing.T) ProcessHealth {
 		return WAITING
 	}
 
+	if strings.Contains(line, "error getting file stat for readLinked file") {
```

Comment:
Match for all kind of errors, not the only that one, in other words we need to verify that during startup we didn't encounter any single error